src/test/java/redis/clients/jedis/tests/commands/AllKindOfValuesCommandsTest.java

```diff
@@ -488,6 +490,23 @@ public void dumpAndRestore() {
     assertTrue(jedis.exists("foo2"));
   }
 
+  @Test(expected = JedisBusyException.class)
```

Comment:
JedisBusyException is more accurate but using JedisException is ok .
no relations to time.